mobile/examples/super_resolution/android/app/build.gradle

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+    id 'kotlin-android-extensions'
+}
+
+android {
+    namespace 'ai.onnxruntime.example.superresolution'
+    compileSdk 32
+
+    defaultConfig {
+        applicationId "ai.onnxruntime.example.superresolution"
+        minSdk 24
+        targetSdk 32
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+    buildFeatures {
+        viewBinding true
+    }
+}
+
+dependencies {
+    implementation 'androidx.core:core-ktx:1.7.0'
+    implementation 'androidx.appcompat:appcompat:1.5.1'
+    implementation 'com.google.android.material:material:1.7.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    testImplementation 'junit:junit:4.13.2'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.4'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.0'
+
+    // Note: may want to update to latest version of ort mobile aar package
+    implementation 'com.microsoft.onnxruntime:onnxruntime-android:1.12.1'
```

Comment:
wondering should be update the ort-android package version here as well? say using the latest version

Reply:
we do that for the other android sample:
https://github.com/microsoft/onnxruntime-inference-examples/blob/a9c13afda3535b22447c1c55a125b00016dca064/mobile/examples/image_classification/android/app/build.gradle#L58

Reply:
cool thanks. will include the same thing here also then.